packages/cli/test/sim/multi_fork.test.ts

```diff
@@ -79,10 +81,14 @@ env.tracker.register({
 await env.start({runTimeoutMs});
 await connectAllNodes(env.nodes);
 
-// The `TTD` will be reach around `start of bellatrixForkEpoch + additionalSlotsForMerge` slot
-// We wait for the end of that epoch with half more epoch to make sure merge transition is complete
-await waitForSlot(env.clock.getLastSlotOfEpoch(bellatrixForkEpoch) + activePreset.SLOTS_PER_EPOCH / 2, env.nodes, {
-  silent: true,
+let lastForkEpoch = 0;
+for (const fork of env.forkConfig.forksAscendingEpochOrder) {
```

Comment:
Please add a nice comment above this code explaining the purpose of this and motivation

Reply:
Done